m: 10.0
Ixx: 0.16
Iyy: 0.16
Izz: 0.16
d_lin: [-4.03, -6.22, -5.18, -0.07, -0.07, -0.07]
d_quad: [-18.18, -21.66, -36.99, -1.55, -1.55, -1.55]
W: 98.1
B: 100.06
z_g: 0.0
z_b: 0.02
neutral_bouy: False

tam: [[0.7071, 0.7071, -0.7071, -0.7071, 0.0, 0.0],
      [-0.7071, 0.7071, -0.7071, 0.7071, 0.0, 0.0],
      [0.0, 0.0, 0.0, 0.0, -1.0, -1.0],
      [0.0, 0.0, 0.0, 0.0, 0.115, -0.115],
      [0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
      [-0.1773, 0.1773, -0.1773, 0.1773, 0.0, 0.0]]

m_added: [-5.50, -12.70, -14.57, -0.12, -0.12, -0.12]